homeassistant/components/avreceiver/__init__.py

```diff
@@ -0,0 +1,172 @@
+"""The AV Receiver integration."""
+from pyavreceiver import const as avr_const, factory
+import voluptuous as vol
+
+from homeassistant.components.media_player import DOMAIN as MEDIA_PLAYER_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, EVENT_HOMEASSISTANT_STOP
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import entity_registry as er
+import homeassistant.helpers.config_validation as cv
+
+from .const import (
+    CONF_ZONE1,
+    CONF_ZONE2,
+    CONF_ZONE3,
+    CONF_ZONE4,
+    CONTROLLER,
+    DOMAIN,
+    SIGNAL_AVR_UPDATED,
+    UNSUB_UPDATE_LISTENER,
+    ZONES,
+)
+
+CONFIG_SCHEMA = vol.Schema(
+    {DOMAIN: vol.Schema({vol.Required(CONF_HOST): cv.string})}, extra=vol.ALLOW_EXTRA
+)
+
+PLATFORMS = [MEDIA_PLAYER_DOMAIN]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the AV Receiver component."""
+    if DOMAIN not in config:
+        return True
+    host = config[DOMAIN][CONF_HOST]
+    entries = hass.config_entries.async_entries(DOMAIN)
+    if not entries:
+        # Create new entry based on config
+        hass.async_create_task(
+            hass.config_entries.flow.async_init(
+                DOMAIN, context={"source": "import"}, data={CONF_HOST: host}
+            )
+        )
```

Comment:
I would just skip this for a new integration.

Reply:
I agree, config import is not needed since YAML is depricated anyway